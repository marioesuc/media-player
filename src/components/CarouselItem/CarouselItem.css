.CarouselItem-container {
  height: auto;
  padding: 15px;
  background-color: rgba(34, 46, 68, 0.5);
  text-align: center;
  font-weight: bold;
  margin: 0px 10px;
  box-shadow: 1px 1px 1px 1px rgba(0, 0, 0, 0.2);
  transition: 0.3s;
}

.CarouselItem-container:hover {
  background-color: orange;
  transition: 0.3s;
}

.CarouselItem-image {
  width: 150px;
}

.CarouselItem-title {
  margin-top: 20px;
}

.link {
  color: white;
  text-decoration: none;
}
