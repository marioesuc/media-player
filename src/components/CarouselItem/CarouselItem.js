import React from "react";
import "./CarouselItem.css";

class CarouselItem extends React.Component {
  render() {
    const { title, imgSrc } = this.props;

    return (
      <div className="CarouselItem-container">
        <img alt="Cover" className="CarouselItem-image" src={imgSrc} />
        <div className="CarouselItem-title">{title}</div>
      </div>
    );
  }
}

export default CarouselItem;
