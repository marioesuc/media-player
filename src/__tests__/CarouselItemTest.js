import React from "react";
import { shallow, configure } from "enzyme";
import CarouselItem from "../components/CarouselItem/CarouselItem";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("CarouselItem", () => {
  const props = {
    title: "title",
    imgSrc: "url_path"
  };

  it("renders correctly", () => {
    const wrapper = shallow(<CarouselItem {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
