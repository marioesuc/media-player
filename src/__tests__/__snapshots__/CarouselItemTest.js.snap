// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarouselItem renders correctly 1`] = `ShallowWrapper {}`;
